src/main/java/fr/insalyon/creatis/gasw/execution/GaswMonitor.java

```diff
@@ -135,8 +135,6 @@ protected void verifySignaledJobs() {
             }
             // Kill job replicas
             for (Job job : jobDAO.getJobs(GaswStatus.KILL_REPLICA)) {
-                job.setStatus(GaswStatus.CANCELLED_REPLICA);
```

Comment:
I wonder if we don't need to update the status asap so that we avoid any possible replication/rescheduling of the job (e.g., by the healing mechanism)